app/src/main/res/layout/activity_edit_profile.xml

```diff
@@ -75,6 +75,34 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/edit_user_last_name" />
 
+    <TextView
+        android:id="@+id/fingerprint_auth_registration_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="40dp"
+        android:text="Fingerprint authentication:"
```

Comment:
we should always avoid hardcoding text

Reply:
Yes, I have already addressed this in the next PR I will be creating just after this one